### DISCUSSION TITLE
Mechanic: "upvote bots"

### DISCUSSION THREAD

**Discussion by PotentialStyx** [cursorweb/Cycle-Bot-Game#19] [Ideas]
New mechanic idea, this one is unique, it has a chance to backfire spectacularly, so it can either: help you out, or basically wipe all your cycle bot progress (back to either the beginning or your last prestige, whichever was close (it doesn't wipe prestige effect or count)).

This mechanic would allow you to buy and upgrade "upvote bots". Each bot will cost about the same as the bot before, but ONLY because of their possible negative effects. 

## upgrades
Their would be 3 ways to upgrade the bot:
- stealth (reduces chance to get caught)
- secrecy (reduces chance if caught that the bot can be traced to you)
- smart (can link with other smart bots to strategically lower sus)

Each path could have `x`  amount of upgrades, buying any upgrade would cost more then the last.

## getting caught

first time getting caught you would get a warn, have all bots without smart upgraded to at least 2 get deleted, and 2x the upvotes you got from them get taken away (you cannot goto negative cycles, you will just be set to 0)

second time is a ban :O, you revert to the start or your last prestige (whichever is more recent), and everything is gone (except prestige), the warn goes away with the ban, the higher any of your bots smart upgrade is, it can have increasing chance to go unnoticed. 

## what the bots actually do
They can increase the amount of cycles you get per post depending on factors including bot upgrades.

# Some thoughts
- this might need extreme balancing
- needs funny messages for being caught
- Maybe bots are only available after prestige 1, and the more prestige's you go the lower the base odds are for a bot to get caught/identified.

**Comment by PotentialStyx** [comment]
@cursorweb check my idea out kthx

**Reply by cursorweb** [reply]
hmm the concept is interesting
but the name definitely needs a change lol
how about: `generator`

remember upvoting bots also might mean top.gg

**Reply by PotentialStyx** [reply]
> hmm the concept is interesting
> but the name definitely needs a change lol
> how about: `generator`
> 
> remember upvoting bots also might mean top.gg

Hmmm good point, the name needs to change

**Reply by cursorweb** [reply]
> > hmm the concept is interesting
> > but the name definitely needs a change lol
> > how about: `generator`
> > remember upvoting bots also might mean top.gg
> 
> Hmmm good point, the name needs to change

but like basically this is a cpp upgrade just buffed and with a risk?

**Reply by PotentialStyx** [reply]
> > > hmm the concept is interesting
> > > but the name definitely needs a change lol
> > > how about: `generator`
> > > remember upvoting bots also might mean top.gg
> > 
> > 
> > Hmmm good point, the name needs to change
> 
> but like basically this is a cpp upgrade just buffed and with a risk?

I guess, but adding a risk kinda changes the whole feel and use of the mechanic, for example: it might help users try to get cycles quick, but it might also reset them. It might help higher level users get more cycles over long times, and also having upgrades could reduce the risk.

**Reply by PotentialStyx** [reply]
It's an item that has a use for low and high players, both sort of differently
